api/aws-cognito-env.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is this the config info just for local development?

Reply:
I'm assuming this will be overwritten by a config map then for each environment?

Reply:
That's the one for local yes. Haven't add it into config map yet. It's in my thought.

Reply:
It is now configured to use FAM PROD - fom_dev, fom_test and fom_prod at /api/openshift.deploy.yml and merge-main.yml.
Please verify the correctness.